config/schema.yaml

```diff
@@ -643,6 +643,18 @@ variables:
       - stage:
           global:
             X_FRAME_CONFIGURATION: SAMEORIGIN
+  QUALITY_PATCHES:
+    description: Specify a list of Magento quality patches that will be applied during deployment. Patches should be
+      separated by space.
+    type: string
+    stages:
+      - build
+    default:
+      build: ''
+    examples:
+      - stage:
+          build:
+            QUALITY_PATCHES: 'MC-31387 MDVA-4567 MC-45634'
```

Comment:
Yeah, I thought about it, but CLI command suppose to take a wildcard as well
`./vendor/bin/ece-patches apply --patches="*"` to apply all optional patches. It was a decision to not advertise the possibility of using wildcard since we don't want to encourage customers to apply a big amount of patches. But we need it internally to run CI build with all patches applied (should guaranty that it's possible to apply all patches without conflicts).

Reply:
I reconsidered how the list of patches should pass in CLI command, changed from option to argument
`./vendor/bin/ece-patches apply --patches="MC-1111 MC-2222"` => `./vendor/bin/ece-patches apply MC-1111 MC-2222`. So now QUALITY_PATCHES has an array type. Now it looks better. Thanks.

Reply:
@viktym maybe we could name it quality-patches instead of just patches to be more explicit? There are chances we can do current required patches optional later via similar argument, so would be good to adjust for that kind of situation.

Reply:
@YPyltiai right now there is no option name, the list of patches passes as argument of the command:
`./vendor/bin/ece-patches apply MC-1111 MC-2222 MDVA-4356`
If we need to make some of the patches from magento/magento-cloud-patches optional, we can move them to magento/quality-patches

Reply:
@viktym that makes sense. We can proceed as is. Thanks for clarification.